js/levels.js

```diff
@@ -164,7 +164,7 @@ var levels = [
       'tr': 'Grid çizgilerini soldan değil, sağdan saymak istiyorsanız, <code>grid-column-start</code> ve <code>grid-column-end</code> özelliklerine negatif değer verebilirisiniz. Örneğin değeri -1 yaparak, sağdan ilk grid çizgisini belirtebilirisiniz.</p><p> <code>grid-column-end</code> özelliğine negatif bir değer eklemeyi deneyin.</p>',
       'fr': '<p>Si vous souhaitez compter les lignes de la grille à partir de la droite plutôt que de la gauche, vous pouvez donner des valeurs négatives à <code>grid-column-start</code> et à <code>grid-column-end</code>. Par exemple, vous pouvez le définir à -1 pour spécifier la première ligne de la grille en partant de la droite.</p><p>Essayez de définir <code>grid-column-end</code> avec une valeur négative.</p>',
       'de': '<p>Falls du die Rasterlinien von rechts statt von links zählen willst, kannst du für <code>grid-column-start</code> und <code>grid-column-end</code> negative Werte einstellen. Du kannst sie zum Beispiel auf -1 setzen um die erste Rasterlinie von rechts anzusprechen.</p><p>Versuche, <code>grid-column-end</code> auf einen negativen Wert zu setzen.</p>',
-      'nl': '<p>Als je grid lijnen vanaf rechts in plaats van links wilt tellen, kan je <code>grid-column-start</code> en <code>grid-column-end</code> negatieve waardes geven. Bijvoorbeeld, je kan hem op -1 zetten om de eerste grid lijn vanaf rechts aan te spreken.</p><p>Probeer <code>grid-column-end</code> een negatieve waarde te geven.</p>',
+      'nl': '<p>Als je gridlijnen vanaf rechts in plaats van links wilt tellen, kan je <code>grid-column-start</code> en <code>grid-column-end</code> negatieve waardes geven. Bijv. kan je hem op -1 zetten om de eerste gridlijn vanaf rechts aan te spreken.</p><p>Probeer <code>grid-column-end</code> een negatieve waarde te geven.</p>',
```

Comment:
In my opinion it's better to keep `Bijvoorbeeld` in stead of `Bijv.` since makes reading more restless.